### ISSUE TITLE
Renaming file to suffix (1) deletes this file

### ISSUE THREAD

**Issue by adi-dev** [nextcloud/desktop#3479]
I just found that if I rename a file for example from `DSC_1234.jpg` to `DSC_1234.jpg (1)` or `DSC_1234.jpg (2)` (and so on) it causes the file to disappear from local folder, server and is not even in deleted on the server. Using virtual files.
Desktop Client 3.2.2 on Windows. Log file is empty.

**Comment by FlexW** [comment]
Are the files downloaded or not when doing the rename?

**Comment by adi-dev** [comment]
The files were already on the local machine, I found that they were processed copies, so I decided to rename and copy the originals in place. Before I copied the new ones, I found that the one I renamed disappeared. When I renamed them to `DSC_1234_fix.jpg` everything worked as expected. Renaming to "_(1).jpg" or so always results in deleting. I can copy a file ending with "(1)" to the same location with no issues. Only when I rename, this happens.
